wp-content/themes/heideandcook/web/sass/_fancybox.scss

```diff
@@ -84,3 +77,14 @@ form > .gform_footer > .gform_button.button:hover {
 .fancybox-skin {
     border-radius: 0px;
 }
+
+.gform_wrapper:not(.ginput_container_multiselect) select {
+    background-image: url();
+}
+
+.gform_body > .gform_wrapper > [type='text'], select, input:not(.button) {
```

Comment:
this is selecting all select elements and inputs that are not .button including ones outside of gravity forms. you want to narrow the scope of the last two selectors.